acquisition_left:
  ros__parameters:
    serialNumber: "KE0220030137"
    exposureTime: 10000
    autoExposure: True # Mporoun na mpoun kai True kai exposureTime, ta exw kanei handle sthn
    # klash ths cameras, ola komple
    expectedGrayValue: 50
    ROIx: 1280
    ROIy: 1024
    orientation: "left"
    nodeTimer: 3.0

acquisition_right:
  ros__parameters:
    serialNumber: "KE0220040196"
    exposureTime: 10000
    autoExposure: True
    expectedGrayValue: 50
    ROIx: 1280
    ROIy: 1024
    orientation: "right"
    nodeTimer: 3.0

# acquisition_center:
#   ros__parameters:
#     serialNumber: "KE0220030138"
#     exposureTime: 10000
#     autoExposure: True
#     expectedGrayValue: 50
#     ROIx: 1280
#     ROIy: 1024
#     orientation: "center"
#     nodeTimer: 3.0 

    # Na vazete double times (float, dhladh gia osous den kseroun, kinhths ypodiastolhs kai OXI integer, giati komplarei to ROS)